app/settings.php

```diff
@@ -89,6 +89,17 @@
                         'donorEmail'
                     ],
                 ],
+                [
+                    'templateKey' => 'password-reset-requested',
+                    'subject' => 'Reset your password for Big Give',
+                    'subjectParams' => [],
+                    'requiredParams' => [
+                        'first_name',
```

Comment:
Hey @bdsl, if possible can you please follow the naming conventions as per the other template definitions above?
In other words:
1. `first_name` should be renamed as `donorFirstName`
2. `last_name` should be renamed as `donorLastName`
3. `recipientEmailAddress` should be renamed as `donorEmail`

Please also ensure their usage is updated in the twig file.

Reply:
Yes, will fix. I think my mistake was doing the identity side first before looking at the context here, and then it seemed easier to follow what I had in identity.

Reply:
@alihejazi fixed in 8dcd355 . As discussed not putting in the word donor because this email has nothing in it specific to the recipient being a donor and we may want to use it for other people in future.

Reply:
And made matching changes in the ID PR.